terracotta-kit/src/assemble/server/bin/start-tc-server.sh

```diff
@@ -54,48 +54,46 @@ function setPluginClasspath {
 
     for pluginDir in "${PLUGIN_LIB_DIR}" "${PLUGIN_API_DIR}"
     do
-        if [ -e "${pluginDir}" ]
+        if [ -d "${pluginDir}" ]
         then
             for jarFile in "${pluginDir}"/*.jar
             do
-                PLUGIN_CLASSPATH=${PLUGIN_CLASSPATH}:${jarFile}
-            done
-#  Adding SLF4j libraries to the classpath of the server to 
-#  support services that may use SLF4j for logging
-            for jarFile in "${TC_SERVER_DIR}"/lib/slf4j*.jar
-            do
-                PLUGIN_CLASSPATH=${PLUGIN_CLASSPATH}:${jarFile}
+                PLUGIN_CLASSPATH="${PLUGIN_CLASSPATH}:${jarFile}"
             done
+
         fi
     done
+    #  Adding SLF4j libraries to the classpath of the server to
```

Comment:
I moved this out of the loop since it has no obvious relationship to either the loop or the conditional, and will cause the slf4j jars to be added twice